grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -536,6 +536,7 @@ class OrderService {
                     productPrice.price = packagePrice
                     productPackage.productPrice = productPrice
                 }
+                productPackage.lastUpdated = new Date()
```

Comment:
I think this should be updated automatically by GORM

Reply:
@awalkowiak It's not working without this line.

Reply:
Weird I would have expected it to be updated as well. I guess it's fine but we should investigate.